test/functional/wallet_migration.py

```diff
@@ -111,7 +112,9 @@ def migrate_and_get_rpc(self, wallet_name, **kwargs):
         # Migrate, checking that rescan does not occur
         with self.master_node.assert_debug_log(expected_msgs=[], unexpected_msgs=["Rescanning"]):
             migrate_info = self.master_node.migratewallet(wallet_name=wallet_name, **kwargs)
-        return migrate_info, self.master_node.get_wallet_rpc(wallet_name)
+        rpc = self.master_node.get_wallet_rpc(wallet_name)
+        assert_equal(rpc.getwalletinfo()["descriptors"], True)
```

Comment:
88b67c95cbc9c3ae75ccbc39e4aba215b9910b8b: Since you're checking the "descriptors" field into `migrate_and_get_rpc`, you could also assert the wallet is sqlite and then remove the duplicated code along the test. e.g.:

```py
        _, multisig0 = self.migrate_and_get_rpc("multisig0")
        assert_equal(multisig0.getwalletinfo()["descriptors"], True)
        self.assert_is_sqlite("multisig0")
```

```py
        _, multisig0 = self.migrate_and_get_rpc("multisig0")
        assert_equal(multisig0.getwalletinfo()["descriptors"], True)
        self.assert_is_sqlite("multisig0")
```

```py
        _, multisig1 = self.migrate_and_get_rpc("multisig1")
        assert_equal(multisig1.getwalletinfo()["descriptors"], True)
        self.assert_is_sqlite("multisig1")
```

Reply:
Done